rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/JacksonCustomizations.java

```diff
@@ -1000,11 +995,6 @@ public static abstract class VulnerabilityApiDTOMixin extends VulnerabilityApiDT
 
         @JsonInclude(JsonInclude.Include.NON_EMPTY)
         @JsonIgnoreProperties({
-                "assessorContactPerson",
```

Comment:
why you removed these 5 fields, as I'm not able to see any of these fields in response structure.

Reply:
Thanks for your comment. I updated response structure.